src/main/java/org/lsc/jndi/JndiServices.java

```diff
@@ -1137,69 +1139,25 @@ public Map<String, LscDatasets> doGetAttrsList(final String base,
 		constraints.setReturningObjFlag(true);
 
 		try {
-			boolean requestPagedResults = false;
-
-			List<Control> extControls = new ArrayList<Control>();
-
-			if (pageSize > 0) {
-				requestPagedResults = true;
-				LOGGER.debug("Using pagedResults control for {} entries at a time", pageSize);
-			}
-
-			if (requestPagedResults) {
-				extControls.add(new PagedResultsControl(pageSize, Control.CRITICAL));
-			}
-			
-			if(sortedBy != null) {
-			    extControls.add(new SortControl(sortedBy, Control.CRITICAL));
-			}
-
-			if (extControls.size() > 0) {
-				ctx.setRequestControls(extControls.toArray(new Control[extControls.size()]));
-			}
-
-			byte[] pagedResultsResponse = null;
```

Comment:
I have no time to test this, but the previous lines adding extended controls seem quite useful, aren't they?
Why did you remove them?

Reply:
I will re-introduce the sortedBy code in the constructor. It appears to be global configuration for each request that is made to LDAP regardless of via script or otherwise.